packages/core/src/js/tracing/span.ts

```diff
@@ -154,3 +154,33 @@ export function addDefaultOpForSpanFrom(client: Client): void {
     }
   });
 }
+
+export const SPAN_THREAD_ID = 'thread.id';
+export const SPAN_THREAD_ID_MAIN = 1;
```

Comment:
Depending on the scenario and where the value is read from `sentry-cocoa` sets MAIN_ID as `0` or `259`.

I don't know the situation on Android, but I think we should leave the IDs for now until we establish how to read them so they map correctly to what the native SDKs are reporting.

Reply:
Makes sense @krystofwoldrich 👍 
Removed the thread id with 91632023dfd92652a1d2391741901ecadf4a12c0 and I'll investigate how to get this information from the native SDKs separately